dockerfiles/web/Dockerfile-test

```diff
@@ -38,6 +38,10 @@ RUN touch /usr/src/app/yarn-error.log
 COPY yarn.lock /usr/src/app/
 RUN chmod -R 777 /usr/src/app/ /var/lib/xkb /.cache /.yarn /.mozilla
 
+# Get Rust; NOTE: using sh for better compatibility with other base images
+RUN curl https://sh.rustup.rs -sSf | sh -s -- -y
+ENV PATH="/root/.cargo/bin:${PATH}"
```

Comment:
Addresses the issue of this build: https://github.com/lucyparsons/OpenOversight/actions/runs/10675835536/job/29588444176